fabtests/benchmarks/benchmark_shared.c

```diff
@@ -292,6 +305,12 @@ int bandwidth_rma(enum ft_rma_opcodes rma_op, struct fi_rma_iov *remote)
 					rx_seq++;
 
 			} else {
+				if (ft_check_opts(FT_OPT_VERIFY_DATA | FT_OPT_ACTIVE)) {
```

Comment:
How is the data checked in rma path?

Reply:
for write with CQ data, there is an CQ generated on the target side, and target called `bw_rx_comp` get the CQ entry. 

data verification happened inside `bw_rx_comp`.

Reply:
what about read/write?

Reply:
They do not have data verification yet.

As the title of PR suggests, it added data verification to some bandwidth tests.